client/k8s.go

```
@@ -235,9 +235,65 @@ func (k K8sClient) GetcStorPVCs(node string) map[string]*util.Volume {
 			PVC:                     *i.Spec.Source.PersistentVolumeName,
 			CSIVolumeAttachmentName: i.Name,
 			AttachementStatus:       util.CheckVolAttachmentError(i.Status),
-			AccessMode:              util.CheckIfAccessable(i),
+			AccessMode:              k.GetPV(*i.Spec.Source.PersistentVolumeName).Spec.AccessModes,
 		}
 		volumes[vol.PVC] = vol
 	}
 	return volumes
 }
+
+// GetPV used to get pvc information
+func (k K8sClient) GetPV(name string) *corev1.PersistentVolume {
+
+	vol, err := k.cs.CoreV1().PersistentVolumes().Get(name, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("Error while getting volume %s", name)
```

The actual error is swallowed. 
We can do something like 
```
klog.Errorf("Error while getting volume %s: {%s}", name,err.Error())
```

---

Similarly, at other places this fix is required.